osu.Game/Graphics/UserInterfaceV2/OsuPopover.cs

```diff
@@ -58,6 +60,14 @@ protected override void PopOut()
             this.FadeOut(fade_duration, Easing.OutQuint);
         }
 
+        protected override bool OnKeyDown(KeyDownEvent e)
```

Comment:
Can you explain why these changes were required? I can't immediately see why from your commit history, and it looks unrelated to the location you're actually changing things (gameplay doesn't use any popovers to my knowledge).

Reply:
[776b60f](https://github.com/ppy/osu/pull/22079/commits/776b60f3b3b133ecd41bc284e727e333c670f949) made global actions handle first in `ManualInputManagerTestScene`s. It made a popover test fail, and was fixed alongside.

The test failed because the popover never expired due to the global action using `Hide()` instead of `this.HidePopover()` from the `OnKeyDown` implementation. I fixed that but also removed the handling from `OnKeyDown` similar to `FocusedTextBox`.